### ISSUE TITLE
Create Reddit bot equivalent of our twitter @borrowbot

### ISSUE THREAD

**Issue by mekarpeles** [internetarchive/openlibrary#4935]

**Comment by navneetsaluja** [comment]
I am working on this.

**Comment by mekarpeles** [comment]
@navneetsaluja do you mind updating this issue w/ some of the details you sent me over slack? Thank you!

**Comment by navneetsaluja** [comment]
Right, so we are aiming to create a Reddit bot for OpenLibrary. We have a couple of ways of going about this:
- we create a custom bot of openlibrary similar to https://www.reddit.com/user/goodreads-bot
- we partner with an existing bot, namely https://www.reddit.com/user/Reddit-Book-Bot/ since its the most popular, except that it only gives pdfs of the ebooks it has in its catalogue and not information about them.

Also, since Goodreads is retiring its' API, I do feel that the first one is better. Please let me know which one do you prefer. 

Features already done: 
- BorrowBot functions added and made basically a Reddit equivalent of BorrowBot

TODO: 
- Make a reddit account called openlibrary-bot and register for the API
- Similar to what [goodreads-bot](https://www.reddit.com/user/goodreads-bot) does, show the metadata of the book along with some links to whether the book is borrowable or not. Unfortunately, the search in OL is iffy and so I was thinking of googling "Book name + ISBN" and then fuzzy searching through OL for the right book? 
- Contact moderators of subreddit if they would be willing to let our bot run on the subreddit (this ideally should be after the first version of our bot is released)

**Comment by mekarpeles** [comment]
@navneetsaluja is there a certain place we should be looking for this PR? Is this in https://github.com/internetarchive/openlibrary-bots?

**Comment by navneetsaluja** [comment]
@mekarpeles Apologies, I have not pushed the PR yet. I'll push it tomorrow after refining the code a bit.

**Comment by mekarpeles** [comment]
hi @navneetsaluja I think this issue has gone cold so I wanted to throw some heat on it and see if the PR you worked on is ready for a pair of eyes or if it may make more sense for someone else to give it a shot :)

**Comment by maifeeulasad** [comment]
Willing to work on this one!

**Comment by mekarpeles** [comment]
This might improve engagement, but it's probably not where we should be spending our time right now (which is on creating value and making sure that value connects with patrons)
